src/UI/Implementation/Component/Dropdown/Renderer.php

```diff
@@ -10,6 +10,12 @@
 use ILIAS\UI\Implementation\Render\ResourceRegistry;
 
 class Renderer extends AbstractComponentRenderer {
+
+	/**
+	 * @var string
+	 */
+	private $class_name = 'dropdown';
```

Comment:
This is a hard nut as we just discussed orally. My solution to this would be to use inside this component here just the default version of the popover and life with the fact, that a designer has no handle to style the dropdown here individually without actually manipulation the php renderer.